ServiceConfig.cs

```diff
@@ -5,13 +5,21 @@ namespace Laserfiche.Repository.Api.Client.Sample.ServiceApp
 {
     internal class ServiceConfig
     {
+        internal const string ACCESS_KEY = "ACCESS_KEY";
+        internal const string SERVICE_PRINCIPAL_KEY = "SERVICE_PRINCIPAL_KEY";
+        internal const string REPOSITORY_ID = "REPOSITORY_ID";
+        internal const string APISERVER_USERNAME = "APISERVER_USERNAME";
```

Comment:
Do we need this APISERVER prefix for these three fields? as the other fields don't have it.

Reply:
I think we added the APISERVER prefix to make it clear that these environment variables are used for Self-Hosted API Server